### DISCUSSION TITLE
HourMode for TimePicker field

### DISCUSSION THREAD

**Discussion by RohanJdhv97** [filamentphp/filament#6785] [Help]
Hi, I just want to know if, is there any property or rule that I can apply in the time-picker field to make it take time in 12 hr format.

**Comment by PseudoDevs** [comment]
![image](https://github.com/filamentphp/filament/assets/28098260/feba85e1-a3fe-483d-b9de-0e4e2b90dfd9)

you can use ->type('time') to achieve this

![image](https://github.com/filamentphp/filament/assets/28098260/72746088-060f-406a-86f6-794fd139e6db)
